docs/installation-aks.md

```diff
@@ -0,0 +1,407 @@
+
+1. Configure kubectl to use the new cluster
+    ```console
+    az aks get-credentials --resource-group aks-group --name aks-cluster
+    ```
+
+1. Verify your cluster is up and running
+    ```console
+    kubectl get nodes
+    ```
+
+---
+
+## Cluster Configuration
+
+Before you can install OSBA onto your cluster, you will first need to install Service Catalog. Service Catalog uses etcd for data persistence and it is *strongly* recommended that you create an etcd cluster with backup and recovery capabilities. We recommend using [etcd operator](https://github.com/coreos/etcd-operator) configured with backup and recovery via Azure Blob Storage for this purpose. The following section provides guidance on how to configure etcd operator on your cluster and then how to install Service Catalog.
+
+### Install Helm
+
+We will use Helm to install etcd Operator, Service Catalog and OSBA. In order to install Helm in your cluster, use the `helm` CLI:
+
+```console
+helm init
+```
+
+### Install etcd
+
+#### Create Storage Account
+
+In order to configure etcd operator to use Azure Blob Storage for backup and recovery purposes, you will need to first create a storage account:
+
+
+First, save your desired account name to an environment variable. It will be reused a few times.
+ **Bash**
+ ```console
+export AZURE_STORAGE_ACCOUNT=<STORAGE ACCOUNT>
+ ```
+
+**PowerShell**
+```console
+$env:AZURE_STORAGE_ACCOUNT = "<STORAGE ACCOUNT>"
+```
+
+Now use the az cli to create a new storage account.
+
+**Bash**
+```console
+az storage account create -n $AZURE_STORAGE_ACCOUNT -g aks-group
+```
+
+**PowerShell**
+```console
+az storage account create -n $AZURE_STORAGE_ACCOUNT -g aks-group
+```
+
+Once the account has been created, retrieve the keys.
+
+**Bash**
+```console
+az storage account keys list -n $AZURE_STORAGE_ACCOUNT -g aks-group -o table
+```
+
+**Powershell**
+```console
+az storage account keys list -n $env:AZURE_STORAGE_ACCOUNT -g aks-group -o table
+```
+
+Save the storage account key to an environment variable for later use.
+ **Bash**
+ ```console
+export AZURE_STORAGE_KEY=<STORAGE_KEY>
+ ```
+
+**PowerShell**
+```console
+$env:AZURE_STORAGE_KEY = "<STORAGE_KEY>"
+```
+
+Next, you will need to create a container for backup storage.
+
+**Bash**
+```console
+az storage container create --name etcd-backups --account-name $AZURE_STORAGE_ACCOUNT --account-key $AZURE_STORAGE_KEY
+```
+
+**PowerShell**
+```console
+az storage container create --name etcd-backups --account-name $env:AZURE_STORAGE_ACCOUNT --account-key $env:AZURE_STORAGE_KEY
+```
+
+#### Install etcd Operator
+
+Service Catalog requires etcd to persist state for everything it manages. We recommend using etcd Operator to install and manage an etcd cluster for Service Catalog. We have included sample values.yaml file to setup etcd operator with a version that supports Azure backup. 
+
+```console
```

Comment:
super mega nit: For command blocks that just have a command and no `$` and output, don't specify `console` so that the command prints in black. This better matches how the command blocks look when you do use `$` along with `console` so that commands are always in black, and console output is always in blue on the doc page.

it just helps people visually scan the page.